.pre-commit-config.yaml

```diff
@@ -6,10 +6,6 @@ repos:
     -   id: check-json
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
-        exclude: |
```

Comment:
I'm not sure why but this reverts the change I introduced on `main`.

Reply:
Good catch, I probably messed up when solving the issues while rebasing.